diff --git a/git.c b/git.c
index 2b5f0b3..1fe7f53 100644
--- a/git.c
+++ b/git.c
@@ -21,7 +21,7 @@ static void gitMain (const git_uint8 * game, git_uint32 gameSize, git_uint32 cac
 
     // Set various globals.    
     gPeephole = 1;
-    gDebug = 0;
+    gDebug = 1;
     
     // Load the gamefile into memory
     // and initialise undo records.
diff --git a/terp.c b/terp.c
index 94ae833..761c25f 100644
--- a/terp.c
+++ b/terp.c
@@ -57,6 +57,9 @@ float git_powf(float x, float y);
 // -------------------------------------------------------------
 // Functions
 
+git_uint32 glulxPC = 0;
+git_uint32 glulxOpcode = 0;
+
 void startProgram (size_t cacheSize, enum IOMode ioMode)
 {
     Block pc; // Program counter (pointer into dynamically generated code)
@@ -85,9 +88,6 @@ void startProgram (size_t cacheSize, enum IOMode ioMode)
     git_uint32 protectPos = 0;
     git_uint32 protectSize = 0;
     
-    git_uint32 glulxPC = 0;
-    git_uint32 glulxOpcode = 0;
-
     acceleration_func accelfunc;
 
     // Initialise the code cache.
